diagnostic.md

```diff
@@ -31,9 +31,28 @@ Between the comments below, write HTML to create a basic web page (using the
 format described in the instructions). The page should have a title in the head
 and an unordered list (with three items) inside the body.
 
-<!-- your answer starts here -->
+<!DOCTYPE html>
+<html>
+<header>
+ <h1>Title goes here</h1>
+ <link rel="stylesheet" type="text/css" href="css/style.css">
+  <script src="tbd.js" type="text/javascript" charset="utf-8" defer></script>
+  </header>
+
+<body>
+<div>
+  <ul>
+    <li> List Item 1 </li>
+    <li> List Item 2 </li>
+    <li> List Item 3 </li>
+  </ul>
+</div>
+</body>
+
+<footer>
+</footer>
 
-<!-- your answer ends here -->
+</html>
```

Comment:
I think you are confusing header with `<head>`. Also, the footer goes inside the body. Inside the `head` tag, you would have a `<title>` tag that you would place your title within.

```HTML
<html>
	<head>
		<title>My Title</title>
	</head>
	<body>
		<ul>
			<li>Item 1</li>
			<li>Item 2</li>
			<li>Item 3</li>
		</ul>
   <footer>
      <p>Some text here</p>
   </footer>
	</body>
</html>
```